app/src/views/private/components/comment-input.vue

```diff
@@ -119,7 +118,7 @@ watch(
 const saving = ref(false);
 const showMentionDropDown = ref(false);
 
-const searchResult = ref<User[]>([]);
+const searchResult = ref<{ first_name: string; last_name: string; email: string; id: string; avatar: string }[]>([]);
```

Comment:
Shouldn't we use `Pick<User, 'id' | 'email' | 'first_name' | 'last_name' | 'avatar'>` here like it's done in `comment-item.vue`?
That way we would detect the current issue on line 33 where
`avatarSource(user.avatar)` shoud actually be `avatarSource(user.avatar.id)` 😉

Reply:
Yeah we should 👍🏻